### ISSUE TITLE
Adapt to latest Wazuh

### ISSUE THREAD

**Issue by jesuslinares** [wazuh/wazuh-ansible#61]

**Comment by SitoRBJ** [comment]
Hello team,

To update the Elastic Stack version at least we must update the following files:

- Elasticsearch

1. ansible-role-elasticsearhc/defaults/main.yml (https://github.com/wazuh/wazuh-ansible/blob/master/ansible-role-elasticsearch/defaults/main.yml), where the field "elastic_stack_version" should be changed.

- Kibana

1. ansible-role-kibana/defaults/main.yml (https://github.com/wazuh/wazuh-ansible/blob/master/ansible-role-kibana/defaults/main.yml), where the fields "elastic_stack_version" and "wazuh_version"  should be changed. 

- Logstash

1. ansible-role-logstash/defaults/main.yml (https://github.com/wazuh/wazuh-ansible/blob/master/ansible-role-logstash/defaults/main.yml), where the field "elastic_stack_version" should be changed.

The tasks of each playbook will be responsible for attaching the version at the time of installation by taking it from these files. 

We must take into account the task files in case the Elastic Stack repositories are modified, for example:
`https://artifacts.elastic.co/packages/6.x/apt stable main`

We also have to consider changes to the template files and configuration files if there are any changes to those files, for example: 
`ansible-role-elasticsearch/templates/wazuh-elastic6-template-alerts.json.j2`
`ansible-role-logstash/templates/01-wazuh.conf.j2`

Regards,

Alfonso

**Comment by SitoRBJ** [comment]
Hello team, 

We have tested the installation of Elastic Stack with the updates in Ubuntu 18.
- Elasticsearch ->6.4.0
- Logstash -> 6.40
- Kibana -> 6.40 

We found a indent problem, once the installation was resolved it was a success. 

The problem was in ansible-role-elasticsearch/task/Debian.yml

```
    - name: Debian/Ubuntu | Oracle Java 8 installer
      apt:
        name: oracle-java8-installer
        state: present
        cache_valid_time: 3600
      tags: install
```

Additionally we have installed the Wazuh agent in Ubuntu 18. 

- Wazuh-Agent -> 3.6

We will proceed to create the PR that updates the repository. 

Regards,

Alfonso
